src/main/java/Storage.java

```diff
@@ -0,0 +1,44 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Represents a Storage Class which is used to handle all
+ * file handling task - writing, deleting, updating.
+ */
+
+public class Storage {
+
+    private File taskFile;
+    private String filePath;
```

Comment:
Either taskFile or filePath should be used instead of "duke.txt" in Storage methods.